website_multi_theme/models/website.py

```diff
@@ -25,11 +26,14 @@ class Website(models.Model):
         comodel_name='website.theme',
         domain=[("asset_ids.view_id", "!=", False)],
         help="Multiwebsite-compatible theme for this website",
+        require=True,
```

Comment:
Please don't add require fields in inherited classes, as this will make that tests that create records from this model fail when this dependent module is not loaded. Instead, put the require on the view.

Reply:
No, it's the same. The sequence is the following in MQT:

- This module is installed along with all the testable ones and its dependencies.
- Then the SQL NOT NULL constrained is added.
- On the second pass, modules are being tested while being loaded.
- Any module adding a website will have the not null error until this module is loaded.

Reply:
I got it. You are right. 
It seems to be an odoo framework issue. Is there a way to fix it in odoo core?